File: complete/src/main/java/org/skinny/drive_to_work/Episode.java
package org.skinny.drive_to_work;

public class Episode {
    private int number;
    private String title;
    private String description;
    private String url;

    private Episode() {

    }

    public Episode(int number, String title, String description, String url) {
        this.number = number;
        this.title = title;
        this.description = description;
        this.url = url;
    }

    public int getNumber() {
        return number;
    }

    public String getTitle() {
        return title;
    }

    public String getDescription() {
        return description;
    }

    public String getUrl() {
        return url;
    }
}

File: complete/src/main/kotlin/org/skinny/drive_to_work/Episode.kt
package org.skinny.drive_to_work;

import com.fasterxml.jackson.databind.annotation.JsonSerialize
import java.time.LocalDate

data class Episode(val id: Int, val title: String,
                   @field:JsonSerialize(using = LocalDateSerializer::class) val releaseDate: LocalDate,
                   val description: String, val url: String) {
    private constructor() : this(0, "", LocalDate.of(2016, 2, 29), "", "")
}
